early-dev/app.R

```diff
@@ -136,12 +134,10 @@ data <- within(data, {
     )
 })
 
-# set datanames
-datanames <- c("ADSL", "ADAE", "ADCM", "ADEX", "ADTR", "ADTRWF", "ADRS", "ADRSSWIM", "ADLB")
-datanames(data) <- datanames
-
 # set join keys
-join_keys(data) <- default_cdisc_join_keys[datanames] # get default keys by name
+join_keys(data) <- default_cdisc_join_keys[
+  c("ADSL", "ADAE", "ADCM", "ADEX", "ADTR", "ADTRWF", "ADRS", "ADRSSWIM", "ADLB")
```

Comment:
Maybe it's better just to exclude some names, instead of including so many : P ?

Reply:
I think the idea was to use all the default datanames for the datasets used in the app. We might be able to drop a few of them if they are not used in some module, but I doubt it.